operator-kustomize/deployment.yaml

```diff
@@ -18,6 +18,9 @@ spec:
         - name: minio-operator
           image: minio/k8s-operator:v3.0.5
           imagePullPolicy: IfNotPresent
+          resources:
+            requests:
+              memory: 64Mi
```

Comment:
I would suggest choosing 256MiB to be safe @dvaldivia

Reply:
we see it using only 11Mb on average, but we can go to 256MiB, it would be loose